python/CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
 include(AddMLIRPython)
 
```

Comment:
This is load bearing: https://github.com/google/iree/blob/f4c4cced32aa6d3db17f85cf86fc3b43836f22ad/llvm-external-projects/iree-dialects/python/CMakeLists.txt#L5